.eslintrc.js

```diff
@@ -12,5 +12,7 @@ module.exports = {
     ],
   },
   ignorePatterns: ['temp', '**/dist', 'samples'],
-  rules: {},
+  rules: {
+    'typescript-eslint/restrict-template-expressions': 'off',
```

Comment:
What's the reason for disabling this - false positives? Could you see if you still see false positive with the latest version (https://github.com/sourcegraph/code-intel-extensions/pull/384)?

Reply:
Oops.